code/script.js

```diff
@@ -28,26 +40,223 @@ const showMessage = (message, sender) => {
+  setTimeout(() => askSubtractionQuestion(), 1000)
+}
+//If you choose multiplication
+const handleMultiplication = () => {
+  mathType = "multiplication"
+  showMessage(`Multiplication`, "user")
+  setTimeout(() => askMultiplicationQuestion(), 1000)
+}
+//If you choose division
+const handleDivision = () => {
+  mathType = "division"
+  showMessage(`Division`, "user")
+  setTimeout(() => askDivisionQuestion(), 1000)
+}
+//Prints a mathproblem with addition
+const askAdditionQuestion = () => {
+  a = Math.floor (Math.random() * 100) + 1
+  b = Math.floor(Math.random() * 100) + 1
+  showMessage (`What is ${a}+${b}?`, "bot")
+  inputWrapper.innerHTML = `
+    <form id="answerForm"> 
+      <label>Answer</label>
+      <input id="answer" type="text" />  
+      <button id="button" class="send-btn" type="submit"> 
+          Send
+      </button>
+    </form>
+  `
+  document.getElementById("answerForm").addEventListener("submit", handleAnswer)
+}
+//Prints a mathproblem with subtraction
+const askSubtractionQuestion = () => {
+  a = Math.floor (Math.random() * 100) + 1
+  b = Math.floor(Math.random() * 100) + 1
+  if (a >= b) {
+    showMessage (`What is ${a}-${b}?`, "bot")
+    subtractOrder = "a-b"
+  } else {
+    showMessage (`What is ${b}-${a}?`, "bot")
+    subtractOrder = "b-a"
+  }
+  inputWrapper.innerHTML = `
+    <form id="answerForm"> 
+      <label>Answer</label>
+      <input id="answer" type="text" />  
+      <button id="button" class="send-btn" type="submit"> 
+          Send
+      </button>
+    </form>
+  `
+  document.getElementById("answerForm").addEventListener("submit", handleAnswer)
+}
+//Prints a mathproblem with multiplication
+const askMultiplicationQuestion = () => {
+  a = Math.floor (Math.random() * 11)
+  b = Math.floor(Math.random() * 11)
+  showMessage (`What is ${a}*${b}?`, "bot")
+  inputWrapper.innerHTML = `
+    <form id="answerForm"> 
+      <label>Answer</label>
+      <input id="answer" type="text" />  
+      <button id="button" class="send-btn" type="submit"> 
+          Send
+      </button>
+    </form>
+  `
+  document.getElementById("answerForm").addEventListener("submit", handleAnswer)
+}
+//Prints a mathproblem with division
+const askDivisionQuestion = () => {
+  a = Math.floor (Math.random() * 10) + 1
+  b = Math.floor(Math.random() * 11)
+  c = a*b
+  showMessage (`What is ${c}/${a}?`, "bot")
+  inputWrapper.innerHTML = `
+    <form id="answerForm"> 
+      <label>Answer</label>
+      <input id="answer" type="text" />  
+      <button id="button" class="send-btn" type="submit"> 
+          Send
+      </button>
+    </form>
+  `
+  document.getElementById("answerForm").addEventListener("submit", handleAnswer)
+}
+//Handle the answer
+let handleAnswer = (addInput) => {
+  addInput.preventDefault()
+  showMessage(`${answer.value}`, "user")
+  setTimeout(() => checkAnswer(), 1000)
+}
+//Checks the answer and replies "correct" or "wrong"
+const checkAnswer = () => {
```

Comment:
This function too, I think could be refactored a bit to be align more to the DRY principle. If you have time, give it a think 😄